### ISSUE TITLE
Docker Quick Start Error

### ISSUE THREAD

**Issue by fresaelens** [afourmy/eNMS#154]
Hi,

I am trying to get eNMS running using the Docker quick start commands. 

I've used both methods (SQLite & PostgreSQL).

I've tried releases 3.4.16 & 3.4.17 but in all cases I get the following error on the docker_enms_1 container:

```[2019-08-09 13:38:41 +0000] [1] [DEBUG] Current configuration:
  config: gunicorn.py
  bind: ['0.0.0.0:5000']
  backlog: 2048
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 3000
  graceful_timeout: 3000
  keepalive: 2
  limit_request_line: 0
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: -
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: True
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: True
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7faa1ace41e0>
  on_reload: <function OnReload.on_reload at 0x7faa1ace42f0>
  when_ready: <function WhenReady.when_ready at 0x7faa1ace4400>
  pre_fork: <function Prefork.pre_fork at 0x7faa1ace4510>
  post_fork: <function Postfork.post_fork at 0x7faa1ace4620>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7faa1ace4730>
  worker_int: <function WorkerInt.worker_int at 0x7faa1ace4840>
  worker_abort: <function WorkerAbort.worker_abort at 0x7faa1ace4950>
  pre_exec: <function PreExec.pre_exec at 0x7faa1ace4a60>
  pre_request: <function PreRequest.pre_request at 0x7faa1ace4b70>
  post_request: <function PostRequest.post_request at 0x7faa1ace4bf8>
  child_exit: <function ChildExit.child_exit at 0x7faa1ace4d08>
  worker_exit: <function WorkerExit.worker_exit at 0x7faa1ace4e18>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7faa1ace4f28>
  on_exit: <function OnExit.on_exit at 0x7faa1acfd0d0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
  raw_paste_global_conf: []
[2019-08-09 13:38:41 +0000] [1] [INFO] Starting gunicorn 19.9.0
[2019-08-09 13:38:41 +0000] [1] [DEBUG] Arbiter booted
[2019-08-09 13:38:41 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)
[2019-08-09 13:38:41 +0000] [1] [INFO] Using worker: sync
[2019-08-09 13:38:41 +0000] [8] [INFO] Booting worker with pid: 8
[2019-08-09 13:38:41 +0000] [1] [DEBUG] 1 workers
[2019-08-09 13:38:42 +0000] [8] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/app.py", line 6, in <module>
    app = create_app(Path.cwd(), environ.get("ENMS_CONFIG_MODE", "Default"))
  File "/eNMS/__init__.py", line 104, in create_app
    register_modules(app)
  File "/eNMS/__init__.py", line 27, in register_modules
    controller.init_app(app)
  File "/eNMS/controller/base.py", line 224, in init_app
    self.fetch_version()
  File "/eNMS/controller/base.py", line 237, in fetch_version
    with open(self.path / "package.json") as package_file:
FileNotFoundError: [Errno 2] No such file or directory: '/package.json'
[2019-08-09 13:38:42 +0000] [8] [INFO] Worker exiting (pid: 8)
[2019-08-09 13:38:42 +0000] [1] [INFO] Shutting down: Master
[2019-08-09 13:38:42 +0000] [1] [INFO] Reason: Worker failed to boot.
[2019-08-09 13:38:50 +0000] [1] [DEBUG] Current configuration:
  config: gunicorn.py
  bind: ['0.0.0.0:5000']
  backlog: 2048
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 3000
  graceful_timeout: 3000
  keepalive: 2
  limit_request_line: 0
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: -
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: True
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: True
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f8792d291e0>
  on_reload: <function OnReload.on_reload at 0x7f8792d292f0>
  when_ready: <function WhenReady.when_ready at 0x7f8792d29400>
  pre_fork: <function Prefork.pre_fork at 0x7f8792d29510>
  post_fork: <function Postfork.post_fork at 0x7f8792d29620>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f8792d29730>
  worker_int: <function WorkerInt.worker_int at 0x7f8792d29840>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f8792d29950>
  pre_exec: <function PreExec.pre_exec at 0x7f8792d29a60>
  pre_request: <function PreRequest.pre_request at 0x7f8792d29b70>
  post_request: <function PostRequest.post_request at 0x7f8792d29bf8>
  child_exit: <function ChildExit.child_exit at 0x7f8792d29d08>
  worker_exit: <function WorkerExit.worker_exit at 0x7f8792d29e18>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f8792d29f28>
  on_exit: <function OnExit.on_exit at 0x7f8792d420d0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
  raw_paste_global_conf: []
[2019-08-09 13:38:50 +0000] [1] [INFO] Starting gunicorn 19.9.0
[2019-08-09 13:38:50 +0000] [1] [DEBUG] Arbiter booted
[2019-08-09 13:38:50 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)
[2019-08-09 13:38:50 +0000] [1] [INFO] Using worker: sync
[2019-08-09 13:38:50 +0000] [8] [INFO] Booting worker with pid: 8
[2019-08-09 13:38:50 +0000] [1] [DEBUG] 1 workers
[2019-08-09 13:38:50 +0000] [8] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/app.py", line 6, in <module>
    app = create_app(Path.cwd(), environ.get("ENMS_CONFIG_MODE", "Default"))
  File "/eNMS/__init__.py", line 104, in create_app
    register_modules(app)
  File "/eNMS/__init__.py", line 27, in register_modules
    controller.init_app(app)
  File "/eNMS/controller/base.py", line 224, in init_app
    self.fetch_version()
  File "/eNMS/controller/base.py", line 237, in fetch_version
    with open(self.path / "package.json") as package_file:
FileNotFoundError: [Errno 2] No such file or directory: '/package.json'
[2019-08-09 13:38:50 +0000] [8] [INFO] Worker exiting (pid: 8)
[2019-08-09 13:38:51 +0000] [1] [INFO] Shutting down: Master
[2019-08-09 13:38:51 +0000] [1] [INFO] Reason: Worker failed to boot.```

**Comment by afourmy** [comment]
Thanks it should be fixed in the latest release, 3.15.

**Comment by fresaelens** [comment]
Hi,

I stell get the exact same error 

```
[2019-08-13 07:33:25 +0000] [1] [DEBUG] Current configuration:
  config: gunicorn.py
  bind: ['0.0.0.0:5000']
  backlog: 2048
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 3000
  graceful_timeout: 3000
  keepalive: 2
  limit_request_line: 0
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: -
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: True
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: True
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f5d509ce1e0>
  on_reload: <function OnReload.on_reload at 0x7f5d509ce2f0>
  when_ready: <function WhenReady.when_ready at 0x7f5d509ce400>
  pre_fork: <function Prefork.pre_fork at 0x7f5d509ce510>
  post_fork: <function Postfork.post_fork at 0x7f5d509ce620>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f5d509ce730>
  worker_int: <function WorkerInt.worker_int at 0x7f5d509ce840>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f5d509ce950>
  pre_exec: <function PreExec.pre_exec at 0x7f5d509cea60>
  pre_request: <function PreRequest.pre_request at 0x7f5d509ceb70>
  post_request: <function PostRequest.post_request at 0x7f5d509cebf8>
  child_exit: <function ChildExit.child_exit at 0x7f5d509ced08>
  worker_exit: <function WorkerExit.worker_exit at 0x7f5d509cee18>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f5d509cef28>
  on_exit: <function OnExit.on_exit at 0x7f5d509e70d0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
  raw_paste_global_conf: []
[2019-08-13 07:33:25 +0000] [1] [INFO] Starting gunicorn 19.9.0
[2019-08-13 07:33:25 +0000] [1] [DEBUG] Arbiter booted
[2019-08-13 07:33:25 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)
[2019-08-13 07:33:25 +0000] [1] [INFO] Using worker: sync
[2019-08-13 07:33:25 +0000] [8] [INFO] Booting worker with pid: 8
[2019-08-13 07:33:25 +0000] [1] [DEBUG] 1 workers
[2019-08-13 07:33:26 +0000] [8] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/app.py", line 6, in <module>
    app = create_app(Path.cwd(), environ.get("ENMS_CONFIG_MODE", "Default"))
  File "/eNMS/__init__.py", line 104, in create_app
    register_modules(app)
  File "/eNMS/__init__.py", line 27, in register_modules
    controller.init_app(app)
  File "/eNMS/controller/base.py", line 224, in init_app
    self.fetch_version()
  File "/eNMS/controller/base.py", line 237, in fetch_version
    with open(self.path / "package.json") as package_file:
FileNotFoundError: [Errno 2] No such file or directory: '/package.json'
[2019-08-13 07:33:26 +0000] [8] [INFO] Worker exiting (pid: 8)
[2019-08-13 07:33:26 +0000] [1] [INFO] Shutting down: Master
[2019-08-13 07:33:26 +0000] [1] [INFO] Reason: Worker failed to boot.
[2019-08-13 07:33:28 +0000] [1] [DEBUG] Current configuration:
  config: gunicorn.py
  bind: ['0.0.0.0:5000']
  backlog: 2048
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 3000
  graceful_timeout: 3000
  keepalive: 2
  limit_request_line: 0
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: -
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: True
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: True
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fe0f67891e0>
  on_reload: <function OnReload.on_reload at 0x7fe0f67892f0>
  when_ready: <function WhenReady.when_ready at 0x7fe0f6789400>
  pre_fork: <function Prefork.pre_fork at 0x7fe0f6789510>
  post_fork: <function Postfork.post_fork at 0x7fe0f6789620>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fe0f6789730>
  worker_int: <function WorkerInt.worker_int at 0x7fe0f6789840>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fe0f6789950>
  pre_exec: <function PreExec.pre_exec at 0x7fe0f6789a60>
  pre_request: <function PreRequest.pre_request at 0x7fe0f6789b70>
  post_request: <function PostRequest.post_request at 0x7fe0f6789bf8>
  child_exit: <function ChildExit.child_exit at 0x7fe0f6789d08>
  worker_exit: <function WorkerExit.worker_exit at 0x7fe0f6789e18>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fe0f6789f28>
  on_exit: <function OnExit.on_exit at 0x7fe0f67a20d0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
  raw_paste_global_conf: []
[2019-08-13 07:33:28 +0000] [1] [INFO] Starting gunicorn 19.9.0
[2019-08-13 07:33:28 +0000] [1] [DEBUG] Arbiter booted
[2019-08-13 07:33:28 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)
[2019-08-13 07:33:28 +0000] [1] [INFO] Using worker: sync
[2019-08-13 07:33:28 +0000] [8] [INFO] Booting worker with pid: 8
[2019-08-13 07:33:28 +0000] [1] [DEBUG] 1 workers
[2019-08-13 07:33:29 +0000] [8] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/app.py", line 6, in <module>
    app = create_app(Path.cwd(), environ.get("ENMS_CONFIG_MODE", "Default"))
  File "/eNMS/__init__.py", line 104, in create_app
    register_modules(app)
  File "/eNMS/__init__.py", line 27, in register_modules
    controller.init_app(app)
  File "/eNMS/controller/base.py", line 224, in init_app
    self.fetch_version()
  File "/eNMS/controller/base.py", line 237, in fetch_version
    with open(self.path / "package.json") as package_file:
FileNotFoundError: [Errno 2] No such file or directory: '/package.json'
[2019-08-13 07:33:29 +0000] [8] [INFO] Worker exiting (pid: 8)
[2019-08-13 07:33:29 +0000] [1] [INFO] Shutting down: Master
[2019-08-13 07:33:29 +0000] [1] [INFO] Reason: Worker failed to boot.
```
